src/createServer.js

```diff
@@ -4,0 +1,37 @@
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { validate } = require('./validate');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = url.pathname.slice(1);
+    const targetCase = url.searchParams.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = validate(originalText, targetCase);
+
+    if (errors.length > 0) {
+      res.statusCode = 400;
+      res.end(JSON.stringify({ errors }));
+
+      return;
+    }
+
+    const convertedText = convertToCase(originalText, targetCase);
+
+    const result = {
+      targetCase,
+      originalText,
+      ...convertedText,
```

Comment:
The `convertToCase` function is expected to return an object with `originalCase` and `convertedText`. Ensure that these properties are included in the `result` object to match the expected response format .